### ISSUE TITLE
Impossible to access an attribute ("viewModes") on a null variable

### ISSUE THREAD

**Issue by stanislavprokopov** [ryssbowh/craft-themes#6]
I am trying to implement a custom theme, but i am getting an error in commerce product edit page:

```
Impossible to access an attribute ("viewModes") on a null variable.
Twig\Error\RuntimeError: Impossible to access an attribute ("viewModes") on a null variable. in /app/vendor/ryssbowh/craft-themes/src/templates/cp/editproduct.twig:21
Stack trace:
#0 /app/vendor/craftcms/cms/src/helpers/Template.php(101): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), NULL, 'viewModes', Array, 'any', false, false)
#1 /app/storage/runtime/compiled_templates/49/495e815151e615071f8ae6b453fbb75c30f07e5bfb498a4ee2aaada2351e5269.php(82): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), NULL, 'viewModes', Array)
#2 /app/vendor/twig/twig/src/Template.php(394): __TwigTemplate_50fd181b024386b10d6a47da4aef9a30cb487860cc027a1e7847f7f82f950004->doDisplay(Array, Array)
```

Not sure if its related to my theme and smth is missing or the plugin has an error.

**Comment by ryssbowh** [comment]
That's strange, seems your layouts have not been installed yet, you can always reinstall them using the Themes settings.
I've fixed this in 3.1.9 (Craft 3.x) though.

Things have changed in this view for Craft 4 so that will take a bit more time.

**Comment by ryssbowh** [comment]
Those templates have now been removed for Craft 4 : https://github.com/craftcms/cms/issues/11224
